File: PositiveNegativeCheck/PostiveNeg.js
num = parseInt(prompt("Enter number to check: "))

if (isNaN(num)) {
    console.log("Wrong input. Please try enter a valid number")

}
else if (num > 0) {
    console.log(`num = ${num} is a Positive Number.`)
}
else if (num < 0) {
    console.log(`num = ${num} is a Negative Number.`)
}
else {
    console.log(`num = ${num} is a Zero.`)
}

File: PositiveNegativeCheck/PostiveNeg.py
# Check number is Positive or Negative:
# num = int(input("Enter number to check: "))
# if num > 0:
#     print(f'num = {num} is Positive Number.')
# elif num < 0:
#     print(f'num = {num} is Negative Number.')
# elif num == 0:
#     print(f'num = {num} is a Zero.')
# else:
#     print(f'num = {num} is a Wrong Input')


try:
    num = int(input("Enter number to check: "))
    if num > 0:
        print(f'num = {num} is a Positive Number.')
    elif num < 0:
        print(f'num = {num} is a Negative Number.')
    else:
        print(f'num = {num} is Zero.')
except ValueError:
    print("Wrong Input. Please enter a valid number.")

